modules/fbs/fbs.module

```diff
@@ -647,10 +657,6 @@ function fbs_form_profile2_onhold_validate($element, $form_state) {
   $now = date('d-m-Y');
   $now_stamp = strtotime($now);
 
-  if ($from_date_stamp < $now_stamp) {
-    form_set_error('profile_provider_fbs][field_fbs_on_hold', t('Start date must be higher than or equal to today') . ' - ' . $now . '.');
-  }
-
```

Comment:
I understand the situation regarding this validation failing when the user makes unrelated changes to her profile. However I am not a fan of removing this validation. It will allow the user to set a value in the past which we do not want to.

Could we instead only perform this check if  it has been changed by the user - e.g. by checking that the field has a different value than the default value?

By using this approach we also avoid removing the users ability to see previous state.